lighthouse-core/test/report/html/renderer/details-renderer-test.js

```diff
@@ -414,8 +414,8 @@ describe('DetailsRenderer', () => {
       assert.equal(anchorEl.href, 'https://www.example.com/script.js');
       assert.equal(sourceLocationEl.textContent, '/script.js:11:5(www.example.com)');
       assert.equal(sourceLocationEl.getAttribute('data-source-url'), sourceLocation.url);
-      assert.equal(sourceLocationEl.getAttribute('data-source-line'), sourceLocation.line);
-      assert.equal(sourceLocationEl.getAttribute('data-source-column'), sourceLocation.column);
+      assert.equal(sourceLocationEl.getAttribute('data-source-line'), `${sourceLocation.line}`);
```

Comment:
nit: you like this better than String()? was the line limit reached if you did that?

Reply:
> nit: you like this better than String()? was the line limit reached if you did that?

it does go over the limit on the `column` lines, but I also don't have a particular preference between the two